### PR TITLE
Fix right menu links and reduce Bitovi logo size

### PR DESCRIPTION
**Pull request by leoj3n**
Fixes https://github.com/canjs/canjs/issues/2739

### CODE REVIEW

File: static/header.less [hunk 1]
```diff
@@ -33,7 +33,7 @@
             width: 100%;
             cursor: pointer;
             background-image: url(img/canjs_logo.svg);
-            background-size: cover;
+            background-size: 30px;
```
Comment (chasenlehara) [inline static/header.less]: Could you please include before & after screenshots to show what this change does?

File: templates/header-right.mustache [hunk 2]
```diff
@@ -5,7 +5,8 @@
 	    	<ul class="dropdown-menu">
 	      		<li><a href="http://bitovi.com">Bitovi.com</a></li>
 	      		<li><a href="http://bitovi.com/blog/">Blog</a></li>
-	      		<li><a href="http://bitovi.com/consulting/">Consulting</a></li>
+	      		<li><a href="http://bitovi.com/design/">Design</a></li>
+	      		<li><a href="http://bitovi.com/development/">Development</a></li>
```
Comment (chasenlehara) [inline templates/header-right.mustache]: These should be updated to match the destination after redirecting, e.g. https://www.bitovi.com/development

**Comment by leoj3n** [comment]
@chasenlehara Whew, nice catch, I edited the wrong bit of Less.

Before & after:

image

File: templates/header-right.mustache [hunk 3]
```diff
@@ -3,11 +3,12 @@
...
-	      		<li><a href="http://bitovi.com/blog/">Blog</a></li>
-	      		<li><a href="http://bitovi.com/consulting/">Consulting</a></li>
-	      		<li><a href="http://bitovi.com/training/">Training</a></li>
-	      		<li><a href="http://bitovi.com/open-source/">Open Source</a></li>
+	      		<li><a href="https://bitovi.com">Bitovi.com</a></li>
+	      		<li><a href="https://bitovi.com/blog/">Blog</a></li>
+	      		<li><a href="https://bitovi.com/design/">Design</a></li>
+	      		<li><a href="https://bitovi.com/development/">Development</a></li>
+	      		<li><a href="https://bitovi.com/training/">Training</a></li>
+	      		<li><a href="https://bitovi.com/open-source/">Open Source</a></li>
```
Comment (chasenlehara) [inline templates/header-right.mustache]: All of these redirect to a different URL, e.g. https://www.bitovi.com/open-source

**Comment by chasenlehara** [comment]
@leoj3n Heh, yeah, that is the #1 reason why I always try to include screenshots in my PRs: double check that I actually changed what should’ve been changed, and it’s nice for reviewers too. 😊

**Comment by chasenlehara** [comment]
@tomgreever Does the new logo size look good to you?

**Comment by tomgreever** [comment]
👍

**Pull request merged**